### PR TITLE
More checks from Travis

### PR DESCRIPTION
**Pull request by fstagni**
- Added check from travis: pylint, unit tests, format with pep8
- removed docs (everything goes in DIRAC main repo)
- removed dirac-install.py (find it in DIRAC main repo)

### CODE REVIEW

File: runPylint.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+find . -name "*.py" -and -not -name 'pep8_*' -exec pylint -E --rcfile=.pylintrc --msg-template="{path}:{line}: [{msg_id}({symbol}), {obj}] {msg}" {} +
```
Comment (wkrzemien) [inline runPylint.sh]: endline missing

File: Pilot/MessageSender.py [hunk 2]
```diff
@@ -15,6 +15,8 @@
 
 import Queue
 import logging
+import requests
```
Comment (wkrzemien) [inline Pilot/MessageSender.py]: those imports have been hidden, cause not all the nodes have stomp and requests installed.
In principle only RESTMessage sender requires requests, and STOMP Message sender requires stomp.
I don't know if we want to add it explicitely?

File: .travis.yml [hunk 3]
```diff
@@ -2,12 +2,33 @@ language: python
...
+    elif [[ "${CHECK}" == "format" ]] && [[ "${TRAVIS_PULL_REQUEST}" != "false" ]];
+    then git remote add GH https://github.com/DIRACGrid/Pilot.git; git fetch --no-tags GH ${TRAVIS_BRANCH}; git branch -vv; git diff -U0 GH/${TRAVIS_BRANCH} | pycodestyle --diff;
+    fi
+
+## ignore errors for sourcefiles "-i"
 after_success:
-  - coveralls
+  - if [[ "${CHECK}" == "py.test"  ]];
+    then coveralls -i;
+    fi
```
Comment (wkrzemien) [inline .travis.yml]: end line missing

File: Pilot/MessageSender.py (hunk 2 above)
Comment (fstagni) [reply Pilot/MessageSender.py]: OK I re-added the late import of them.

File: Pilot/tests/.pytest_cache/v/cache/lastfailed [hunk 4]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (wkrzemien) [inline Pilot/tests/.pytest_cache/v/cache/lastfailed]: Is this file necessary?

File: PilotUUID [hunk 5]
```diff
@@ -0,0 +1 @@
+55c79850-e98f-11e8-bd73-efcd49162a8b
```
Comment (wkrzemien) [inline PilotUUID]: endline missing

File: TestConf.json [hunk 6]
```diff
@@ -0,0 +1 @@
+{"Setups": {"Dirac-Certification": {"Logging": {"LoggingType": "MQ", "Queue": {"test.cern.ch": {}}, "HostKey": "certificates/client/key.pem", "Url": "", "LocalOutputFile": "", "HostCertificate": "certificates/client/cert.pem", "CACertificate": "certificates/testca/cacert.pem", "Host": "127.0.0.1", "FileWithID": "PilotUUID_test", "Port": "61614"}}}}
```
Comment (wkrzemien) [inline TestConf.json]: endline missing

File: Pilot/tests/.pytest_cache/v/cache/lastfailed (hunk 4 above)
Comment (wkrzemien) [reply Pilot/tests/.pytest_cache/v/cache/lastfailed]: I am not familar with this pytest cache so probably the answer is yes

Comment (andresailer) [reply Pilot/tests/.pytest_cache/v/cache/lastfailed]: No, `.pytest_cache` is a temporary folder created by pytest, none of the files should be kept inside git. (There is even a README file in this folder which says so...)

File: Pilot/tests/.pytest_cache/README.md [hunk 7]
```diff
@@ -0,0 +1,8 @@
+# pytest cache directory #
+
+This directory contains data from the pytest's cache plugin,
+which provides the `--lf` and `--ff` options, as well as the `cache` fixture.
+
+**Do not** commit this to version control.
```
Comment (andresailer) [inline Pilot/tests/.pytest_cache/README.md]: @wkrzemien
